packages/loaders/src/Skeleton.js

```diff
@@ -75,9 +74,9 @@ const StyledSkeleton = styled.div.attrs({
     /* stylelint-disable */
     background-image:
       linear-gradient(
-        -45deg,
+        45deg,
```

Comment:
Doesn't this need to be spun to `-45deg` to retain expected orientation for RTL?